ghactions.py

```diff
@@ -59,11 +59,13 @@ def get_recipient(repo):
 
 
 def parse_payload(run):
+    # Many of the variables below may appear to be unused, but they are passed to the templates via locals()
     job_status = run.get("conclusion", "unknown")
     job_name = run.get("name", "???")
     job_url = run.get("html_url", "")
     job_id = run.get("workflow_id", "")
     job_repo = run.get("repository", {}).get("name", "infrastructure-unknown")
+    job_branch = run.get("TBA", "???") # FIX: how do we get this?
```

Comment:
according to this: https://docs.github.com/en/webhooks/webhook-events-and-payloads#workflow_run
it should be `head_branch`

Reply:
the property might be null so be aware when using it in the templates

Reply:
Something to Keeper in mind: for PRs from forks the head_repo will be different so if you include that Info you need to include that Info as well otherwise it might be wrong for such PRs

Reply:
so you can create a PR for a repo. Workflows related to the PR run in the target repo, which is job_repo. But the branch that related to the workflow run is located in a different repo, head_repository with head_branch.

So when you send a notification that workflow run for `repo/head_branch` has been completed it will not be correct in case of PRs from forks.

You should include head_repository as well, I would change the template like this:

```
The GitHub Actions job "{job_name}" on {job_repo}.git has failed.
Run started by GitHub user {job_actor} (triggered by {job_trigger}).

Ref:
{head_repository} / {head_branch}

Head commit for run:
{trigger_hash} / {trigger_author} <{trigger_email}>
{trigger_log}
```

Reply:
As I suggested to add this information:

```
{head_repository} / {head_branch}
```

to the email then you have all details imho. I would just not append the branch name to the first sentence. Most of the time the `head_repository` will be the same as the repository the workflow runs at, but no always and then it might be confusing.